trio_run_in_process/process.py

```diff
@@ -34,7 +34,11 @@ def __init__(
         self._state_changed = trio.Event()
 
     def __str__(self) -> str:
-        return f"Process[{self._async_fn},args={self._args}]"
+        func = self._async_fn.__name__
+        if self._has_pid.is_set():
+            return f"Process[{func},args={self._args},pid={self.pid}]"
+        else:
+            return f"Process[{func},args={self._args}]"
```

Comment:
*nitpick* maybe something like `f"Process[{func},args={self._args},pid=???]"`.  or `<unknown>` or `<unset>`  maybe as alternatives to `???`

Reply:
Went with `<unset>`